c/Configbump/build-info.json

```diff
@@ -0,0 +1,11 @@
+{
+    "package_name" : "configbump",
+    "github_url":"https://github.com/che-incubator/configbump.git",
+    "version": "v0.1.5",
+    "default_branch": "main",
+    "package_dir": "c/Configbump",
```

Comment:
@shreyakajbaje we expect the package folder will all small case, so please update directory name. `c/configbump`